src/patent_client/epo/ops/register/example_ep_register.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```

Comment:
Hey! Can we delete this and replace with standalone test files?

We need a `api_test.py` file with your API test method, and a `manager_test.py` file with what you've called a model test (but I would call a manager test). 

Take a look at the [manager_test.py](https://github.com/parkerhancock/patent_client/blob/master/src/patent_client/epo/ops/published/manager_test.py) and [api_test.py](https://github.com/parkerhancock/patent_client/blob/master/src/patent_client/epo/ops/published/api_test.py) as examples. All you need is a class or set of functions that runs some bit of code, and then assert statements to check for correct output.